.github/labeler.yml

```
@@ -0,0 +1,16 @@
+feature:
+  - head-branch: ['^feature/']
+dependencies:
+  - head-branch: ['^dependencies/']
+enhancement:
+  - head-branch: ['^enhancement/']
+bug:
+  - head-branch: ['^bug/']
```

```suggestion
  - head-branch: ['^fix/']
```

I think the branch names tend to be more to do with that they do, so this is normally `fix`.